[2024-01-23T22:35:20.997+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_postgres_dag.insert_data_to_postgres_task scheduled__2024-01-09T00:00:00+00:00 [queued]>
[2024-01-23T22:35:21.033+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_postgres_dag.insert_data_to_postgres_task scheduled__2024-01-09T00:00:00+00:00 [queued]>
[2024-01-23T22:35:21.035+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-01-23T22:35:21.061+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): insert_data_to_postgres_task> on 2024-01-09 00:00:00+00:00
[2024-01-23T22:35:21.079+0000] {standard_task_runner.py:60} INFO - Started process 228 to run task
[2024-01-23T22:35:21.087+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'example_postgres_dag', 'insert_data_to_postgres_task', 'scheduled__2024-01-09T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/salvaarquivo.py', '--cfg-path', '/tmp/tmp4ae9xc2j']
[2024-01-23T22:35:21.089+0000] {standard_task_runner.py:88} INFO - Job 35: Subtask insert_data_to_postgres_task
[2024-01-23T22:35:21.168+0000] {task_command.py:423} INFO - Running <TaskInstance: example_postgres_dag.insert_data_to_postgres_task scheduled__2024-01-09T00:00:00+00:00 [running]> on host 728782aa8dd3
[2024-01-23T22:35:21.451+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_postgres_dag' AIRFLOW_CTX_TASK_ID='insert_data_to_postgres_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-09T00:00:00+00:00'
[2024-01-23T22:35:21.528+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T22:35:21.538+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=example_postgres_dag, task_id=insert_data_to_postgres_task, execution_date=20240109T000000, start_date=20240123T223520, end_date=20240123T223521
[2024-01-23T22:35:21.599+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T22:35:21.616+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
